src/python/zigopt/sigoptcompute/adapter.py

```diff
@@ -62,7 +60,7 @@
 
 
 class SCAdapter(Service):
-  def call_sigoptcompute(self, cls, view_input):
+  def call_compute(self, cls, view_input):
```

Comment:
actually fine with the original name of this method.

Reply:
Okay, this was `sed` being overzealous